src/FsAutoComplete.Core/CommandResponse.fs

```diff
@@ -248,6 +264,11 @@ module CommandResponse =
 
   let info (serialize : Serializer) (s: string) = serialize { Kind = "info"; Data = s }
   let error (serialize : Serializer) (s: string) = serialize { Kind = "error"; Data = s }
```

Comment:
I'm happy to break the contract in exchange for the new format being more uniform. I agree with @Krzysztof-Cieslak